src/app/chef/[id]/_common/TopSection.tsx

```diff
@@ -0,0 +1,42 @@
+import Link from "next/link";
+
+import { RecipeAppT10Chefs } from "@/mock";
+
+import { Button } from "@/components/button";
+import { ArrowIcon } from "@/components/icons";
+import { ImageComponent } from "@/components/image";
+
+const followerNumber = 5678;
+
+export const generateStaticParams = () => {
+  return [{ id: "1" }, { id: "2" }, { id: "3" }];
+};
+// https://nextjs.org/docs/app/api-reference/functions/generate-static-params
+// 上記参考に動的ルーティングに対応しておりますが、DBと合わせた際の挙動まで考えて実装していません。
+
+export const TopSection = ({ params }: { params: { id: string } }) => {
+  const { id } = params;
+
+  return (
+    <div>
+      <ImageComponent alt={""} ratio={"1/1"} width={"full"} />
```

Comment:
昔はコンポーネントのpropに{}使ってた気がするんですけど、最近見ないので{}を使わないのが最近の流れな気がします。
これだけ画像を使ったアプリになると通信環境悪い時に画像読み込みがエラーになり得ることはあるのでできるだけaltは指定してください！（つけても意味がないもの意外）

```suggestion
      <ImageComponent alt={`${RecipeAppT10Chefs[0].name}の画像`} ratio="1/1" width="full" />
```